assets/in

```diff
@@ -74,9 +74,6 @@ if [ "$skip_download" = "false" ]; then
     --entrypoint /tmp/print-metadata  \
     "$image_name" > ${destination}/metadata.json
 
-  mkdir -p ${destination}/rootfs/
-  docker export $(cat /tmp/container.cid) | tar -xf - -C ${destination}/rootfs/
```

Comment:
I'm unclear as to what is affected by this. Is the ATC worker part of the task running code? If so why is there a hard requirement here and what are the symptoms of removing this image root?

Looks like it's somehow tied into metadata and version? Thought that would all be encapsulated in the contracts of stdin/out.